sunspot/lib/sunspot/search/stats_json_row.rb

```diff
@@ -0,0 +1,88 @@
+module Sunspot
+  module Search
+    class StatsJsonRow
+      attr_reader :data, :value, :nested
+      attr_writer :instance #:nodoc:
+
+      def initialize(data, facet = nil, value = nil) #:nodoc:
+        @data, @facet, @value = data, facet, value
+        @facet_fields = []
+        @nested_key = data.keys.select { |k| data[k].is_a?(Hash) }.first
+        @nested = recursive_nested_initialization(data) unless @nested_key.nil?
+      end
+
+      def min
+        data['min'].to_i
+      end
+
+      def max
+        data['max'].to_i
+      end
+
+      def count
+        data['count'].to_i
+      end
+
+      def sum
+        data['sum'].to_i
+      end
+
+      def missing
+        data['missing'].to_i
+      end
+
+      def sumsq
+        sum_of_squares
+      end
+
+      def sum_of_squares
+        data['sumsq'].to_i
+      end
+
+      def avg
+        mean
+      end
+
+      def mean
+        data['avg'].to_i
```

Comment:
`mean` loads data from `avg` but is called `mean`. Lets change method name to `avg`. If you want to add an alias method, please use `alias new_name old_name` instead of declaring a new method explicitly. Same for `sum_of_squares` above